module/documents/actor.mjs

```diff
@@ -265,22 +582,22 @@ export default class ActorEd extends Actor {
     return foundry.utils.expandObject( enrichment );
   }
 
-  async _enableHTMLEnrichmentEmbeddedItems( ) {
+  async _enableHTMLEnrichmentEmbeddedItems() {
     for ( const item of this.items ) {
       item.system.description.value = foundry.utils.expandObject( await TextEditor.enrichHTML( item.system.description.value, {
-            async: true,
-            secrets: this.isOwner,
-          } )
+        async: true,
+        secrets: this.isOwner,
+      } )
       );
     }
   }
 
   async addLpTransaction( type, transactionData ) {
     const oldTransactions = this.system.lp[type];
-    const transactionModel = type === "earnings" ? LpEarningTransactionData : LpSpendingTransactionData
-    const transaction = new transactionModel( transactionData )
+    const TransactionModel = type === "earnings" ? LpEarningTransactionData : LpSpendingTransactionData
```

Comment:
Closing semicolons messing in multiple lines

Reply:
can we make it an eslint ?

Reply:
added semi: ["error", "always"] under rules